eng/pipelines/runtimelab/coreclr-tests.yml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+
+pr: none
```

Comment:
why don't want it to be triggered by PR? or CI?

Reply:
It takes 5 hours to complete right now and it's not of a huge value for most changes. Having it on a similar plan as GCStress runs seems ideal. Thanks to the pointer to the issue discussing `pr: none`. I'll try that out!

Reply:
Sounds good. Then I would recommend just removing the `pr: none` from here and then setting up the pipeline accordingly in the UI so that it is optional and only triggered by comments.